packages/jest-test-sequencer/src/__tests__/test_sequencer.test.ts

```diff
@@ -368,6 +368,23 @@ test('returns expected 100/8 shards', async () => {
   );
 
   expect(shards.map(shard => shard.length)).toEqual([
-    13, 13, 13, 13, 13, 13, 13, 9,
+    13, 13, 13, 13, 12, 12, 12, 12,
```

Comment:
somewhat surprising this changed as well?

Reply:
I expected it when implementing. Given the new distribution is strictly better I thought I'd roll with it instead of special-casing the dangling-0 group case.